### DISCUSSION TITLE
Add a "Play on Start" option to the AnimationPlayer

### DISCUSSION THREAD

**Discussion by SaadTheGlad** [godotengine/godot-proposals#8163] [General Discussions]
It would be very convenient to have an option on the AnimationPlayer where when enabled, it automatically plays the animation on start of the game. This would forgo the need for an extra script just to start it, if the object with said animation only needs one for an idle and adding an extra script would just add bloat.

**Comment by jmarceno** [comment]
I believe that if you set the an animation as "Current Animation" on your animation player and then enable "Autoplay on Load" for that animation, it will play on start. This how I did for some animations in the past. 

Autoplay on Load is the small "A" icon just after the animation name on your Animation tab.

**Comment by mieldepoche** [comment]
There is such button, in the animation panel. 

![image](https://github.com/godotengine/godot-proposals/assets/71937306/28544ddd-c9fc-4f9a-87b9-862d6fc40df0)

**Reply by SaadTheGlad** [reply]
Oh wow I am stupid, I didn't see that, thank you.

**Comment by SaadTheGlad** [comment]
The option exists, it is titled Autoplay on Load
